### PR TITLE
Add packages open.mp v1.2.0.2670 and refactor.

### PR DESCRIPTION
**Pull request by MichelDev2005**
Hello...
Now the appropriate alias appears as requested, pointing to the most current version ( @ADRFranklin ).

### CODE REVIEW

File: runtimes.json [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
     "aliases": {
         "latest": "0.3.7-R2-2-1",
-        "openmp": "openmp-rc2",
+        "openmp": "openmp-120",
         "0.3.DL": "0.3.DL-R1",
         "0.3.7": "0.3.7-R2-2-1",
         "0.3z": "0.3z-R4",
-        "UGMP": "UG-MP"
+        "UGMP": "UG-MP",
+        "openmp-rc2": "openmp-rc2",
```
Comment (ADRFranklin) [inline runtimes.json:9]: Remove these aliases as they point to the same version and are not needed, aliases are for refering to changing versionings like pointing to the latest of a specific set of packages

File: runtimes.json [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+        "openmp-rc1": "openmp-rc1"
```
Comment (ADRFranklin) [inline runtimes.json:10]: Same with this one as above

**Comment by MichelDev2005** [comment]
@ADRFranklin - I removed it, but I inform you that RC1 and RC2 are different versions of the current lastest of open.mp, I suggest that we also keep the alias of RC2 to a minimum plus the current one that is in "openmp".

**Comment by ADRFranklin** [comment]
> @ADRFranklin - I removed it, but I inform you that RC1 and RC2 are different versions of the current lastest of open.mp, I suggest that we also keep the alias of RC2 to a minimum plus the current one that is in "openmp".

I am aware, you can still use `openmp-rc2` and `openmp-rc1` because that is what the name of the version is called, aliases link to a specific version name, that is all. 

For example the version name here is `openmp-dynssl` https://github.com/sampctl/runtimes/blob/cbc99c831553dc87f98dd1ce2eea5b0d11d1f821/runtimes.json#L12 
which is name you can reference in your pawn.json to use it, the same allies to every release in that array you can just specify the version name and use it. The point of aliasing is to point to a version and be able to change it without the end user having to manually specify the latest version everytime it comes out, the alias will just point to it and the user will get the newest version that the alias points to right when they ensure again.

**Comment by ADRFranklin** [comment]
I have just noticed that `openmp-120` does not exist in the list itself, you know you have to manually add the release right? like you need to download the binaries for windows/linux and perform a md5 checksum on the archives and then create a new array with the version name and the url to where to download the files with the checksum that matches it.

This is not automated in a way, where specifying the release name found in openmp will let sampctl download it, you have to manually specify where sampctl will fetch the download from

File: runtimes.json [hunk 3]
```diff
@@ -1,15 +1,105 @@
...
+                "components/Recordings.so": "components/Recordings.so",
+                "components/TextDraws.so": "components/TextDraws.so",
+                "components/TextLabels.so": "components/TextLabels.so",
+                "components/Timers.so": "components/Timers.so",
+                "components/Unicode.so": "components/Unicode.so",
+                "components/Variables.so": "components/Variables.so",
+                "components/Vehicles.so": "components/Vehicles.so",
+                "omp-server": "samp03svr"
+            },
+            "win32_paths": null
```
Comment (ADRFranklin) [inline runtimes.json:42]: Remove this, keeping it null will trigger older versions to fail

**Pull request merged**